### ISSUE TITLE
Wrong input 28. with alteon_config_slb_pip6

### ISSUE THREAD

**Issue by gitnetofr** [Radware/radware-ansible#13]
##### SEVERITY
Critical

##### SUMMARY
alteon_config_slb_pip6 module doesn't work.

##### STEPS TO REPRODUCE
```
alteon-sdk==0.6b57
radware-ansible-common-alteon==0.3b35
radware-sdk-common==0.4b32

ansible-galaxy collection version==0.0.6

****************

oalteonpip6:
  - pip6_base_type: vlan
    pip6_vlans: 1103
    pip6_pip6_addr: "1103::1"

****************

- name: Setting up Proxy IP v6
  radware.radware_modules.alteon_config_slb_pip6:
    state: present
    parameters:
      base_type: "{{ item.pip6_base_type | default(omit) }}"
      vlans: "{{ item.pip6_vlans | default(omit) }}"
      ports: "{{ item.pip6_ports | default(omit) }}"
      pip6_addr: "{{ item.pip6_pip6_addr | default(omit) }}"
    provider: "{{ ConnectionParams }}"
  delegate_to: localhost
  register: __outputPip6Deploy
  with_items: "{{ pip6 }}"

```

##### EXPECTED RESULTS
```
/c/slb/pip/type vlan
/c/slb/pip/add6 1103:0:0:0:0:0:0:1 1103
```

##### ACTUAL RESULTS

```
ansible_collections.radware.radware_modules.plugins.module_utils.common.RadwareModuleError:
Rest Request Error:
Status: 406
Reason: None
url= https://xxx:443/config/Pip6NewCfgTable/1103::1
method= PUT
body= {
"AddVlan":"1103",
}
Details: {
         "status":"err",
         "testErr":"Wrong input 28.",
         "mibName": "Pip6NewCfgTable,AddVlan"
}.
Alteon
[WARNING]: please verify your alteon is running a version >= ['31.0.10.0', '32.2.2.0']
failed: [alteon_32.4.1.10] (item={'pip6_base_type': 'vlan', 'pip6_vlans': 1103, 'pip6_pip6_addr': '1103::1'}) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "parameters": {
                "base_type": "vlan",
                "pip6_addr": "1103::1",
                "ports": null,
                "vlans": [
                    1103
                ]
            },
            "provider": {
                "https_port": 443,
                "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
                "server": "xxx",
                "ssh_port": 22,
                "timeout": 20,
                "user": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
                "validate_certs": false
            },
            "revert_on_error": false,
            "state": "present",
            "write_on_change": false
        }
    },
    "item": {
        "pip6_base_type": "vlan",
        "pip6_pip6_addr": "1103::1",
        "pip6_vlans": 1103
    },



    "msg": "\nRest Request Error:\nStatus: 406\nReason: None\nurl= https://xxx:443/config/Pip6NewCfgTable/1103::1\nmethod= PUT\nbody= {\n\"AddVlan\":\"1103\",\n}\nDetails: {\n\t \"status\":\"err\",\n\t \"testErr\":\"Wrong input 28.\",\n\t \"mibName\": \"Pip6NewCfgTable,AddVlan\"\n}.\nAlteon"

```

**Comment by rdwrtac** [comment]
Hi, can you please provide your mail address so we can log the case in Radware system?
Thanks.

**Comment by rdwrtac** [comment]
This issue is resolved in 32.6.2.0. Released and can be found [https://portals.radware.com/Customer/Home/Downloads/Application-Delivery-Load-Balancing/?Product=Alteon](url)
